transport/errors.go

```diff
@@ -0,0 +1,6 @@
+package transport
+
+import "errors"
+
+// ErrNoTransportAvailable occurs when all available transports fail to report output, or if no transports were configured.
```

Comment:
**[golangci-lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
line is 123 characters (lll)